src/thor/triplegbuilder.cc

```diff
@@ -286,6 +286,7 @@ TripLeg_Node_Type GetTripLegNodeType(const NodeType node_type) {
     case NodeType::kBorderControl:
       return TripLeg_Node_Type_kBorderControl;
   }
+  throw std::runtime_error("Unhandled case");
```

Comment:
This is too generic. i know it cant actually happen but we should be more specific about what case is unhandled in the very least